src/webui/webapplication.cpp

```diff
@@ -213,6 +249,21 @@ namespace
                 .arg(scope).arg(actionName[0].toUpper())
                 .arg(actionName.right(actionName.size() - 1));
     }
+
+    void applyToTorrents(const QStringList &hashes, const std::function<void (BitTorrent::TorrentHandle *torrent)> &func)
+    {
+        if ((hashes.size() == 1) && (hashes[0] == QLatin1String("all"))) {
+            foreach (BitTorrent::TorrentHandle *const torrent, BitTorrent::Session::instance()->torrents())
```

Comment:
nitpick, `const` here and below is a bit redundant IMO.

Reply:
Of course. Just a copy-paste here...